improver/wxcode/wxcode_decision_tree.py

```diff
@@ -109,9 +119,7 @@ def wxcode_decision_tree():
             "probability_thresholds": [0.5],
             "threshold_condition": ">=",
             "condition_combination": "",
-            "diagnostic_fields": [
-                "probability_of_lwe_precipitation_rate_above_threshold"
-            ],
+            "diagnostic_fields": [PRECIP_PROB_ABOVE],
```

Comment:
Thanks for the tidy up Caroline. Whilst lightning is only used in one place (line 89 above), is it worth defining it with all the other variables in the __init__ to keep things together? I think it probably is.

Reply:
Oops, sorry missed that one.